source/jsonnet/northern-ireland/individual/blocks/identity-and-health/religion_other.jsonnet

```diff
@@ -0,0 +1,67 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local question(title) = {
+  id: 'religion-other-question',
+  title: title,
+  guidance: {
+    contents: [
+      {
+        description: 'This question is <strong>voluntary</strong>',
+      },
+    ],
+  },
+  type: 'General',
+  answers: [
+    {
+      id: 'religion-other-answer',
+      label: 'Religion',
+      description: 'Enter your own answer or select from suggestions',
+      max_length: 100,
+      mandatory: false,
+      suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/v1.0.0/religions.json',
+      type: 'TextField',
+    },
+  ],
+};
+
+{
+  type: 'Question',
+  id: 'religion-other',
+  question_variants: [
+    {
+      question: question('You selected “Any other religion”. What is your religion?'),
```

Comment:
According to the spreadsheet it should be: `You selected "Other". What religion, religious denomination or body do you belong to?`